Adding onBadInput callback

alexf2 opened the pull request:
There may be a need to know, when user makes a bad input. For example, if you want to highlight the input for a short time.

uhlryk commented:
Thank you for this improvement. It looks great.
I would like to merge it. But I think that it would be great if you would also add few words in README.md about this new functionality.

alexf2 commented:
1. Added some Readme doc for onBadInput.
2. Fixed some wording in the doc.
3. Added an example for onBadInput, showing red highlighting (the most bottom input).
4. Added "run-example" Npm script along with hot-reloading.
5. Made some corrections in package.json  Npm scripts to make them working on Windows.
6. Ported the example to Bootstrap 4.
7. Made some marginal refactorings.

P.S. I suggest testing Npm scripts on Linux or Mac. I didn't as I don't have them at hand.
